src/js/components/Box/StyledBox.js

```diff
@@ -206,6 +206,23 @@ const gapStyle = (directionProp, gap, responsive, wrap, theme) => {
   const responsiveMetric = responsive && breakpoint && breakpoint.edgeSize[gap];
 
   const styles = [];
+  if (typeof gap === 'object') {
```

Comment:
This is looking good. We are missing the `breakPointStyle` that the other gap values have. I think we should add that, can follow pattern of what is done below in this function:
```
if (responsiveMetric) ...
```

Reply:
Ill take a look

Reply:
done